dev/bots/suite_runners/run_add_to_app_life_cycle_tests.dart

```diff
@@ -0,0 +1,23 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io' show Platform;
+
+import 'package:path/path.dart' as path;
+
+import '../run_command.dart';
+import '../utils.dart';
+
+Future<void> addToAppLifeCycleRunner(String flutterRoot) async {
+  if (Platform.isMacOS) {
+    printProgress('${green}Running add-to-app life cycle iOS integration tests$reset...');
+    final String addToAppDir = path.join(flutterRoot, 'dev', 'integration_tests', 'ios_add2app_life_cycle');
+    await runCommand('./build_and_test.sh',
+      <String>[],
+      workingDirectory: addToAppDir,
+    );
+  } else {
+    printProgress('${yellow}Skipped on this platform (only iOS has add-to-add lifecycle tests at this time).$reset');
```

Comment:
Although, how do we even reach this?

Reply:
Does it need to be reached in a different way than the current `SHARD=add_to_app_life_cycle_tests bin/cache/dart-sdk/bin/dart dev/bots/test.dart`? My understanding was that the current commands to trigger this specific suite of tests will still be maintained since the "SHARD" parameter is being handled in the same way in test.dart

Reply:
do we have a non-ios .ci.yaml target configured that calls that? and does it do any work?

Reply:
I looked through all `recipe: flutter/flutter_drone` .ci.yaml targets and don't see any non-ios targets configured with `add_to_app_life_cycle_tests` as a parameter or without a shard at all

Reply:
Can we just throw here then?

Reply:
Yes that makes sense to me

Reply:
Added an exception thrown there